frontend/__mocks__/hooks/api/runtimeData.ts

```diff
@@ -14,6 +14,7 @@ const mockedUseRuntimeData = useRuntimeData as jest.MockedFunction<
 export function getMockRuntimeDataWithPremium(): RuntimeData {
   return {
     FXA_ORIGIN: "https://example.com",
+    BASKET_ORIGIN: "https://basket.mozilla.org",
```

Comment:
Question: Why are we using this URL instead of the development URL? `https://basket-dev.allizom.org`

Reply:
That is a good question; we shouldn't. In fact, it's probably best to not use a realistic URL in the first place, since this is mock data, and we want to make sure we're hitting the values from the `runtime_data` API at the right moment, rather than a fallback value. Fixed in 2a6868af.